src/IntervalArith/Approx/Approx.purs

```diff
@@ -282,6 +282,12 @@ exact (Approx _ _ e _) = e == (big 0)
 
 exact Bottom = false
 
+-- |Returns 'True' if the approximation is finite, i.e., it is not Bottom
+finite :: Approx -> Boolean
```

Comment:
Rename to `isFinite` as it is a boolean check

Reply:
Thank you.
Renamed finite-> isFinite and exact -> isExact